### PR TITLE
LPD-39678 Performance issues deleting object entries

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-39678](https://liferay.atlassian.net/browse/LPD-39678)

[LPD-39678]: https://liferay.atlassian.net/browse/LPD-39678?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:report:57947187

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:report:58803112

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java [hunk 1]
```diff
@@ -68,14 +77,72 @@ public void deleteRelatedModel(
...
 
+			int maxpoolsize = GetterUtil.getInteger(
+				SystemProperties.get("system.executor.service.maxpoolsize"), 2);
+
+			ExecutorService executorService = new ThreadPoolExecutor(
+				maxpoolsize, maxpoolsize);
+
+			List<Future<?>> futures = new ArrayList<>();
+
+			boolean skipObjectEntryResourcePermission =
```
Comment (linolaoj) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java:88]: Hi @pedro-oliveira446 , can you please reorder here?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Review by linolaoj — APPROVED** [review]

**Comment by pedro-oliveira446** [comment]
Resent https://github.com/brianchandotcom/liferay-portal/pull/155833

**Pull request closed without merging**